controllers/xmlrpc/src/main/java/orca/controllers/xmlrpc/OrcaXmlrpcHandler.java

```diff
@@ -1882,7 +1882,12 @@ protected static Properties getEthModifyProperties(String parent_prefix, String
 			// check the whitelist
 			if (verifyCredentials && !checkWhitelist(userDN)) 
 				return setError(WHITELIST_ERROR);
-			
+
+			if (!verifyCredentials && null == userDN){
```

Comment:
I'm concerned this may happen if an invalidly formatted cert is used to connect to the controller. In this case user DN may end up being NULL too and then what happens? There may need to be an additional external flag indicating you are in emulation, then overriding user DN is allowed.